File: LLD_Java/BookMyShow/Theatre/Seat.java
package BookMyShow.Theatre;
import BookMyShow.Model.*;
public class Seat {
    int seatID;
    int rowNum;
    SeatType seatType;
    public int getSeatID(){
        return seatID;
    }
    public void setSeatID(int seatID){
        this.seatID = seatID;
    }
    public int getRowNum(){
        return rowNum;
    }
    public void setRowNum(int rowNum){
        this.rowNum = rowNum;
    }
    public SeatType getSeatType(){
        return seatType;
    }
    public void setSeatType(SeatType seatType){
        this.seatType = seatType;
    }
}

File: LLD(Python)/bookMyShow/Model/Seat.py
class Seat:
    def __init__(self):
        self.__seatID = None
        self._seatType = None
        self._rowno = None

    def setSeatID(self, sid):
        self.__seatID = sid

    def getSeatID(self):
        return self.__seatID
    
    def setSeatType(self, stype):
        self._seatType = stype
    
    def getSeatType(self):
        return self._seatType
    
    def setRowno(self, stype):
        self._rowno = stype
    
    def getRowno(self):
        return self._rowno

    
